### ISSUE TITLE
Cannot import file _gopackagename as it is not generated

### ISSUE THREAD

**Issue by playtime222** [go-python/gopy#246]
This is the start of the stuff.py file generated from stuff.go in the mypackage go package:
```
# python wrapper for package mypackage within overall package mypackage
# This is what you import to use the package.
# File is generated by gopy. Do not edit.
# gopy pkg mypackage

# the following is required to enable dlopen to open the _go.so file
import os,sys,inspect,collections
cwd = os.getcwd()
currentdir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
os.chdir(currentdir)
import _mypackage
os.chdir(cwd)
<snip>
```

There is no _mypackage.py file, only a _mypackage.so.

Error is:
```
  File "<snip>/stuff.py", line 12, in <module>
    import _mypackage
ModuleNotFoundError: No module named '_mypackage'
```
Quite stumped.

**Comment by justinfx** [comment]
It expects to find a `_mypackage.so` in the same directory as that python file. Is that the case? However it could also be related to a problem where gopy is not generating files with python3-style absolute package imports that work if your generated code is in another package. I have a pending merge request to fix that: #245 245

**Comment by rcoreilly** [comment]
assuming this is fixed with @justinfx path fixes.
